dimod/sampleset.py

```diff
@@ -646,7 +650,22 @@ def first(self):
 
     @property
     def info(self):
-        """Dict of information about the :class:`SampleSet` as a whole."""
+        """Dict of information about the :class:`SampleSet` as a whole.
+
+        Examples:
+           This example shows the type of information that might be returned by
```

Comment:
I am worried that this example is too specific. The sample set is used all over the place the info will not always be the same.

Reply:
The intro sentence is pretty strongly worded to that effect: "type of information that might be returned"